test/lexers/jsx-lexer.test.js

```diff
@@ -317,6 +317,20 @@ describe('JsxLexer', () => {
         assert.equal(Lexer.extract(content)[0].defaultValue, 'Some Content')
         done()
       })
+
+      it('output self-closing when transSupportBasicHtmlNodes is true and element is self-closing', (done) => {
+        const Lexer = new JsxLexer({transSupportBasicHtmlNodes: true})
+        const content = '<Trans>a<br />b</Trans>'
+        assert.equal(Lexer.extract(content)[0].defaultValue, 'a<br />b')
+        done()
+      })
+
+      it('do not output self-closing when transSupportBasicHtmlNodes is true but element is not self-closing', (done) => {
+        const Lexer = new JsxLexer({transSupportBasicHtmlNodes: true})
+        const content = '<Trans>a<strong></strong>b</Trans>'
```

Comment:
I don't see a self-closing tag in `content`. Is that intentional?

Reply:
Yes, the tests are to validate that the tag is not rewritten by the parser when there's no child content, both when the developer writes a self-closing or not self-closing tag. Previously a self-closed tag was always outputted as not self-closed, which is the issue I'm trying to solve.